src/util/get-github-client-config.ts

```diff
@@ -123,14 +90,14 @@ const buildGitHubClientServerConfig = async (gitHubServerApp: GitHubServerApp, j
 	}
 );
 
-const buildGitHubClientCloudConfig = async (jiraHost: string, logger: Logger): Promise<GitHubClientConfig> => {
+const buildGitHubClientCloudConfig = async (jiraHost: string): Promise<GitHubClientConfig> => {
```

Comment:
This won't be async too.

Reply:
This one has to because of
`const privateKey = await keyLocator(undefined, jiraHost);`